Sources/WebAuthn/WebAuthnManager.swift

```diff
@@ -139,7 +141,7 @@ public struct WebAuthnManager: Sendable {
         timeout: Duration? = .seconds(60),
         allowCredentials: [PublicKeyCredentialDescriptor]? = nil,
         userVerification: UserVerificationRequirement = .preferred
-    ) throws -> PublicKeyCredentialRequestOptions {
```

Comment:
This `throws` was completely unused. Not sure if it was here defensively or not, but removed it in case it was vestigial.

Reply:
I'm guessing the challenge generator used to throw an error, but I don't think that should be an issue now